frontend/src/components/SearchBar.tsx

```diff
@@ -9,71 +10,74 @@ const SearchBar: React.FC<SearchBarProps> = ({ onSearch }) => {
   const [searchTerm, setSearchTerm] = useState('');
 
   const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
-    setSearchTerm(event.target.value);
+    const searchValue = event.target.value;
+    setSearchTerm(searchValue);
+  };
+
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === 'Enter') {
+      onSearch(searchTerm);
+    }
   };
 
   return (
     <div>
       <div className="flex w-full h-9 justify-center items-start gap-x-4 shrink-0">
 
         {/* search bar */}
-        <div className='flex w-96 h-9 py-2 px-4 items-center gap-4 shrink-0 border-solid border rounded-lg border-[#5F5F5F] bg-[#FFFDFD]'>
-          <svg xmlns="http://www.w3.org/2000/svg" className="h-4 w-4" fill="none" viewBox="0 0 24 24" stroke="#9A9A9A"><path strokeLinecap="round" strokeLinejoin="round" strokeWidth="4" d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z" /></svg>
-          {/* <p className='text-[#9A9A9A] italic text-xs font-normal'>Search Name or Title</p> */}
-          <div className="form-control">
-            <div className="input-group">
+        <div className={`flex w-96 h-9 py-2 px-4 items-center gap-4 shrink-0 border-solid border rounded-lg border-[${colors.dark_gray}] bg-[${colors.white}]`}>
+          <svg xmlns="http://www.w3.org/2000/svg" className="h-4 w-4" fill="none" viewBox="0 0 24 24" stroke={`${colors.med_gray}`}><path strokeLinecap="round" strokeLinejoin="round" strokeWidth="4" d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z" /></svg>
+          <div className="form-control w-full">
```

Comment:
with svg's it could be beneficial to make this into its own svg file and import it into the code for reusability and readability